tests/integration/targets/proxmox/tasks/main.yml

```diff
@@ -0,0 +1,111 @@
+####################################################################
+# WARNING: These are designed specifically for Ansible tests       #
+# and should not be used as examples of how to write Ansible roles #
+####################################################################
+
+# Copyright: (c) 2020, Tristan Le Guern <tleguern at bouledef.eu>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+- name: List domains
+  community.general.proxmox_domain_info:
```

Comment:
FQCN isn't needed here (thanks to some ansible-test magic). These are only needed in tests for plugins that are no action plugins or modules.

Reply:
Fixed. I did not even noticed with the tasks further down the file.